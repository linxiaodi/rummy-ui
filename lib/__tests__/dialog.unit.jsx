import React from 'react';
import { shallow, mount } from 'enzyme';
import Dialog from '../dialog';

const evtFn = jest.fn();

describe('dialog', () => {
	it('render correct', () => {
		const c = mount(<Dialog
			visible={ true }
		>Dialog12</Dialog>);
		expect(c).toMatchSnapshot();
	});

});