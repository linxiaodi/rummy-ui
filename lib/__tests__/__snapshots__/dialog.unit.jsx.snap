// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dialog render correct 1`] = `
<Dialog
  closeOnClickMask={true}
  visible={true}
>
  <Portal
    containerInfo={
      <body
        style="overflow: hidden;"
      >
        <div
          class="ru-dialog-mask"
        />
        <div
          class="ru-dialog"
        >
          <div
            class="ru-dialog-header"
          >
            <div
              class="ru-dialog-header__title"
            />
            <i
              class="ru-icon ru-dialog-close ru-icon-close"
            />
          </div>
          <div
            class="ru-dialog-body"
          >
            Dialog12
          </div>
        </div>
      </body>
    }
  >
    <div
      className="ru-dialog-mask"
      onClick={[Function]}
    />
    <div
      className="ru-dialog"
    >
      <div
        className="ru-dialog-header"
      >
        <div
          className="ru-dialog-header__title"
        />
        <Index
          className="ru-dialog-close"
          name="close"
          onClick={[Function]}
        >
          <i
            className="ru-icon ru-dialog-close ru-icon-close"
            onClick={[Function]}
          />
        </Index>
      </div>
      <div
        className="ru-dialog-body"
      >
        Dialog12
      </div>
    </div>
  </Portal>
</Dialog>
`;

exports[`modal render correct 1`] = `
<Dialog
  closeOnClickMask={true}
  visible={true}
>
  <Portal
    containerInfo={
      <body
        style="overflow: hidden;"
      >
        <div
          class="ru-dialog-mask"
        />
        <div
          class="ru-dialog"
        >
          <div
            class="ru-dialog-header"
          >
            <div
              class="ru-dialog-header__title"
            />
            <i
              class="ru-icon ru-dialog-close ru-icon-close"
            />
          </div>
          <div
            class="ru-dialog-body"
          >
            Dialog12
          </div>
        </div>
      </body>
    }
  >
    <div
      className="ru-dialog-mask"
      onClick={[Function]}
    />
    <div
      className="ru-dialog"
    >
      <div
        className="ru-dialog-header"
      >
        <div
          className="ru-dialog-header__title"
        />
        <Index
          className="ru-dialog-close"
          name="close"
          onClick={[Function]}
        >
          <i
            className="ru-icon ru-dialog-close ru-icon-close"
            onClick={[Function]}
          />
        </Index>
      </div>
      <div
        className="ru-dialog-body"
      >
        Dialog12
      </div>
    </div>
  </Portal>
</Dialog>
`;
